Composer/packages/client/src/pages/design/exportSkillModal/constants.tsx

```diff
@@ -1,11 +1,14 @@
 // Copyright (c) Microsoft Corporation.
 // Licensed under the MIT License.
 
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
```

Comment:
just import React, this comes at a cost that is not necessary.

Reply:
Refactor